### ISSUE TITLE
Memcached PHP Module won't install

### ISSUE THREAD

**Issue by jordy-wijman** [puphpet/puphpet#2271]
I've been trying lots of things already, but nothing seems to work. I added memcached in my php modules list, did a vagrant provision and it just won't work. I've also tried manually adding memcached php to the VM but it's also not working. I've destroyed and made the VM like 5 times, no luck at all.

Oh and preferably I'd run on php 5.6, but since someone else suggested running 5.5 to see if that worked, I tried doing that. But no luck either that way.

Can someone PLEASE help me out?

**config.yaml**
```
vagrantfile:
    target: local
    vm:
        provider:
            local:
                box: puphpet/ubuntu1404-x64
                box_url: puphpet/ubuntu1404-x64
                box_version: '0'
                chosen_virtualizer: virtualbox
                virtualizers:
                    virtualbox:
                        modifyvm:
                            natdnshostresolver1: false
                        showgui: 0
                    vmware:
                        numvcpus: 1
                    parallels:
                        use_linked_clone: 0
                        check_guest_tools: 0
                        update_guest_tools: 0
                machines:
                    vflm_k85tfs6k2anr:
                        id: triple
                        network:
                            private_network: 192.168.56.101
                            forwarded_port:
                                vflmnfp_ueq4iwa6ww4g:
                                    host: '8708'
                                    guest: '22'
                        hostname: triple.dev
                        memory: '512'
                        cpus: '1'
        provision:
            puppet:
                manifests_path: puphpet/puppet/manifests
                module_path: puphpet/puppet/modules
                options:
                    - '--verbose'
                    - '--hiera_config /vagrant/puphpet/puppet/hiera.yaml'
        synced_folder:
            vflsf_4fuwsdfexg73w:
                source: ./
                target: /var/www
                sync_type: default
                smb:
                    smb_host: ''
                    smb_username: ''
                    smb_password: ''
                    mount_options:
                        dir_mode: '0775'
                        file_mode: '0664'
                rsync:
                    args:
                        - '--verbose'
                        - '--archive'
                        - '-z'
                    exclude:
                        - .vagrant/
                        - .git/
                    auto: 'true'
                owner: www-data
                group: www-data
        usable_port_range:
            start: 10200
            stop: 10500
        post_up_message: ''
    ssh:
        host: 'false'
        port: 'false'
        private_key_path: 'false'
        username: vagrant
        guest_port: 'false'
        keep_alive: '1'
        forward_agent: 'false'
        forward_x11: 'false'
        shell: 'bash -l'
        insert_key: 'false'
    vagrant:
        host: detect
    proxy:
        http: ''
        https: ''
        ftp: ''
        no_proxy: ''
        enabled: ''
server:
    install: '1'
    packages:
        - vim
        - git
users_groups:
    install: '1'
    groups: {  }
    users: {  }
locale:
    install: '1'
    settings:
        default_locale: en_US.UTF-8
        locales:
            - en_GB.UTF-8
            - en_US.UTF-8
            - nl_NL.UTF-8
        timezone: Europe/Amsterdam
firewall:
    install: '1'
    rules: {  }
cron:
    install: '1'
    jobs:
        cj_vjiy5ogfi59u:
            name: ''
            user: ''
            command: ''
            minute: ''
            hour: ''
            weekday: ''
            month: ''
            monthday: ''
nginx:
    install: '0'
    settings:
        default_vhost: 1
        proxy_buffer_size: 128k
        proxy_buffers: '4 256k'
        proxy_connect_timeout: 600s
        proxy_send_timeout: 600s
        proxy_read_timeout: 600s
        names_hash_bucket_size: 128
    upstreams: {  }
    vhosts:
        nxv_4b4fhdrvcd7n:
            server_name: awesome.dev
            server_aliases:
                - www.awesome.dev
            www_root: /var/www/awesome
            listen_port: '80'
            index_files:
                - index.html
                - index.htm
                - index.php
            client_max_body_size: 1m
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_port: '443'
            ssl_protocols: ''
            ssl_ciphers: ''
            rewrite_to_https: '1'
            spdy: '1'
            locations:
                nxvl_xlxmizbkafe6:
                    location: /
                    autoindex: 'off'
                    internal: 'false'
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: ''
                    fastcgi_index: ''
                    fastcgi_split_path: ''
                    proxy: ''
                    proxy_redirect: ''
                nxvl_vif1s955fxtr:
                    location: '~ \.php$'
                    autoindex: 'off'
                    internal: 'false'
                    try_files:
                        - $uri
                        - $uri/
                        - /index.php$is_args$args
                    fastcgi: '127.0.0.1:9000'
                    fastcgi_index: index.php
                    fastcgi_split_path: '^(.+\.php)(/.*)$'
                    fast_cgi_params_extra:
                        - 'SCRIPT_FILENAME $request_filename'
                        - 'APP_ENV dev'
                    set:
                        - '$path_info $fastcgi_path_info'
                    proxy: ''
                    proxy_redirect: ''
    proxies: {  }
apache:
    install: '1'
    settings:
        version: 2.4
        user: www-data
        group: www-data
        default_vhost: true
        manage_user: false
        manage_group: false
        sendfile: 0
    modules:
        - proxy_fcgi
        - rewrite
        - ldap
    vhosts:
        av_5fowzvqu0ymj:
            servername: qa.dev
            serveraliases:
                - www.qa.dev
            docroot: /var/www/qa/htdocs
            port: '80'
            setenv:
                - 'APP_ENV dev'
            setenvif:
                - 'Authorization "(.*)" HTTP_AUTHORIZATION=$1'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_4m4ahhtxic9g:
                    path: /var/www/qa/htdocs
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_uwc1citriclj:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9000'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory

        av_5werwzvqu0ymj:
            servername: triple.dev
            serveraliases:
                - www.triple.dev
            docroot: /var/www/triple-website/htdocs
            port: '80'
            setenv:
                - 'APP_ENV dev'
            setenvif:
                - 'Authorization "(.*)" HTTP_AUTHORIZATION=$1'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_4m4ahhtxic9g:
                    path: /var/www/triple-website/htdocs
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_uwc1citriclj:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9000'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory

        av_5fowzvqweymj:
            servername: framework.dev
            serveraliases:
                - www.framework.dev
            docroot: /var/www/framework/htdocs
            port: '80'
            setenv:
                - 'APP_ENV dev'
            setenvif:
                - 'Authorization "(.*)" HTTP_AUTHORIZATION=$1'
            custom_fragment: ''
            ssl: '0'
            ssl_cert: ''
            ssl_key: ''
            ssl_chain: ''
            ssl_certs_dir: ''
            ssl_protocol: ''
            ssl_cipher: ''
            directories:
                avd_4m4ahhtxic9g:
                    path: /var/www/framework/htdocs
                    options:
                        - Indexes
                        - FollowSymlinks
                        - MultiViews
                    allow_override:
                        - All
                    require:
                        - 'all granted'
                    custom_fragment: ''
                    files_match:
                        avdfm_uwc1citriclj:
                            path: \.php$
                            sethandler: 'proxy:fcgi://127.0.0.1:9000'
                            custom_fragment: ''
                            provider: filesmatch
                    provider: directory
php:
    install: '1'
    settings:
        version: '55'
    modules:
        php:
            - cli
            - intl
            - ldap
            - mcrypt
            - curl
            - xmlrpc
            - mbstring
            - memcached
        pear: {  }
        pecl: {  }
    ini:
        display_errors: 'On'
        error_reporting: '-1'
        session.save_path: /var/lib/php/session
        date.timezone: UTC
        open_basedir: 'NULL'
    fpm_ini:
        error_log: /var/log/php-fpm.log
    fpm_pools:
        phpfp_5cpz7z9wtnrt:
            ini:
                prefix: www
                listen: '127.0.0.1:9000'
                security.limit_extensions: .php
                user: www-user
                group: www-data
    composer: '1'
    composer_home: ''
xdebug:
    install: '1'
    settings:
        xdebug.default_enable: '1'
        xdebug.remote_autostart: '0'
        xdebug.remote_connect_back: '1'
        xdebug.remote_enable: '1'
        xdebug.remote_handler: dbgp
        xdebug.remote_port: '9000'
blackfire:
    install: '0'
    settings:
        server_id: ''
        server_token: ''
        agent:
            http_proxy: ''
            https_proxy: ''
            log_file: stderr
            log_level: '1'
        php:
            agent_timeout: '0.25'
            log_file: ''
            log_level: '1'
xhprof:
    install: '0'
wpcli:
    install: '0'
    version: v0.21.1
drush:
    install: '0'
    version: 6.3.0
ruby:
    install: '1'
    versions:
        rv_2qvmfobzs2d6:
            default: '1'
            bundler: '1'
            version: 1.9.3
            gems:
                - deep_merge
                - activesupport
                - vine
python:
    install: '1'
    packages: {  }
    versions: {  }
nodejs:
    install: '0'
    settings:
        version: '5'
    npm_packages: {  }
hhvm:
    install: '0'
    nightly: 0
    composer: '1'
    composer_home: ''
    settings: {  }
    server_ini:
        hhvm.server.host: 127.0.0.1
        hhvm.server.port: '9000'
        hhvm.log.use_log_file: '1'
        hhvm.log.file: /var/log/hhvm/error.log
    php_ini:
        display_errors: 'On'
        error_reporting: '-1'
        date.timezone: UTC
mariadb:
    install: '1'
    settings:
        version: '10.1'
        root_password: '123'
        override_options: {  }
    adminer: 0
    users:
        mariadbnu_7kf6w277ufg6:
            name: dbuser
            password: '123'
    databases:
        mariadbnd_kv9yx3dtbxp8:
            name: qa
            sql: ''
    grants:
        mariadbng_00f1zbjxg0bx:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
mysql:
    install: '0'
    settings:
        version: '5.6'
        root_password: '123'
        override_options: {  }
    adminer: 0
    users:
        mysqlnu_oyxt9mp4dfs0:
            name: dbuser
            password: '123'
    databases:
        mysqlnd_qjle03zm0mo8:
            name: dbname
            sql: ''
    grants:
        mysqlng_vu79sem6ej61:
            user: dbuser
            table: '*.*'
            privileges:
                - ALL
postgresql:
    install: '0'
    settings:
        global:
            encoding: UTF8
            version: '9.4'
        server:
            postgres_password: '123'
    databases: {  }
    users: {  }
    grants: {  }
    adminer: 0
mongodb:
    install: '0'
    settings:
        bind_ip: 127.0.0.1
        port: '27017'
    globals:
        version: 2.6.0
    databases: {  }
redis:
    install: '0'
    settings:
        port: '6379'
sqlite:
    install: '0'
    adminer: 0
    databases: {  }
mailhog:
    install: '0'
    settings:
        smtp_ip: 0.0.0.0
        smtp_port: 1025
        http_ip: 0.0.0.0
        http_port: '8025'
        path: /usr/local/bin/mailhog
beanstalkd:
    install: '0'
    settings:
        listenaddress: 0.0.0.0
        listenport: '11300'
        maxjobsize: '65535'
        maxconnections: '1024'
        binlogdir: /var/lib/beanstalkd/binlog
        binlogfsync: null
        binlogsize: '10485760'
    beanstalk_console: 0
rabbitmq:
    install: '0'
    settings:
        port: '5672'
    users: {  }
    vhosts: {  }
    plugins: {  }
elastic_search:
    install: '0'
    settings:
        version: 2.1.0
        java_install: true
    instances:
        esi_yxrlraejpevt:
            name: es-01
solr:
    install: '0'
    settings:
        version: 4.10.2
        port: '8984'
```

**Comment by adyz** [comment]
I also get: 
```
Error: Execution of '/usr/bin/yum -d 0 -e 0 -y install php-memcached' returned 1: Package libmemcached10 is obsoleted by libmemcached-last, but obsoleting package does not provide for requirements
==> vivre: Error: Package: php-pecl-memcached-2.2.0-2.el6.remi.5.5.x86_64 (remi-php55)
==> vivre:            Requires: libmemcachedutil.so.2()(64bit)
==> vivre:            Available: libmemcached-last-libs-1.0.18-2.el6.remi.x86_64 (remi)
==> vivre:                libmemcachedutil.so.2()(64bit)
==> vivre:            Available: libmemcached-last-libs-1.0.18-6.el6.remi.x86_64 (remi)
==> vivre:                libmemcachedutil.so.2()(64bit)
==> vivre:            Available: libmemcached10-1.0.16-1.ius.centos6.x86_64 (ius)
==> vivre:                libmemcachedutil.so.2()(64bit)
==> vivre:  You could try using --skip-broken to work around the problem
==> vivre:  You could try running: rpm -Va --nofiles --nodigest
==> vivre: Error: /Stage[main]/Puphpet_php/Puphpet::Php::Module[memcached]/Php::Module[memcached]/Package[PhpModule_memcached]/ensure: change from purged to present failed: Execution of '/usr/bin/yum -d 0 -e 0 -y install php-memcached' returned 1: Package libmemcached10 is obsoleted by libmemcached-last, but obsoleting package does not provide for requirements
==> vivre: Error: Package: php-pecl-memcached-2.2.0-2.el6.remi.5.5.x86_64 (remi-php55)
==> vivre:            Requires: libmemcachedutil.so.2()(64bit)
==> vivre:            Available: libmemcached-last-libs-1.0.18-2.el6.remi.x86_64 (remi)
==> vivre:                libmemcachedutil.so.2()(64bit)
==> vivre:            Available: libmemcached-last-libs-1.0.18-6.el6.remi.x86_64 (remi)
==> vivre:                libmemcachedutil.so.2()(64bit)
==> vivre:            Available: libmemcached10-1.0.16-1.ius.centos6.x86_64 (ius)
==> vivre:                libmemcachedutil.so.2()(64bit)
==> vivre:  You could try using --skip-broken to work around the problem
==> vivre:  You could try running: rpm -Va --nofiles —nodigest
```

**Comment by jtreminio** [comment]
Tested on new release, confirmed working.
